MonoGame.Framework/Graphics/States/Blend.cs

```diff
@@ -58,11 +58,11 @@ public enum Blend
         /// </summary>
 		InverseDestinationAlpha,
 	    /// <summary>
-        /// Each component of the color is multiplied by a constant in the <see cref="Microsoft.Xna.Framework.Graphics.GraphicsDevice.BlendFactor"/>.
+        /// Each component of the color is multiplied by a constant in the <see cref="P:Microsoft.Xna.Framework.Graphics.GraphicsDevice.BlendFactor"/>.

```

Comment:
Turns out it needed not only the full name, but the `P` prefix to tell it the ref is a property.